content/en/docs/concepts/cluster-administration/addons.md

```diff
@@ -19,7 +19,7 @@ This page lists some of the available add-ons and links to their respective inst
 * [Antrea](https://antrea.io/) operates at Layer 3/4 to provide networking and security services for Kubernetes, leveraging Open vSwitch as the networking data plane.
 * [Calico](https://docs.projectcalico.org/latest/introduction/) is a networking and network policy provider. Calico supports a flexible set of networking options so you can choose the most efficient option for your situation, including non-overlay and overlay networks, with or without BGP. Calico uses the same engine to enforce network policy for hosts, pods, and (if using Istio & Envoy) applications at the service mesh layer.
 * [Canal](https://projectcalico.docs.tigera.io/getting-started/kubernetes/flannel/flannel) unites Flannel and Calico, providing networking and network policy.
-* [Cilium](https://github.com/cilium/cilium) is a L3 network and network policy plugin that can enforce HTTP/API/L7 policies transparently. Both routing and overlay/encapsulation mode are supported, and it can work on top of other CNI plugins.
+* [Cilium](https://github.com/cilium/cilium) is a networking, observability, and security solution with an eBPF based dataplane. It provides a simple flat Layer 3 network with the ability to span multiple clusters in either a native routing or overlay/encapsulation mode, and can enforce network policies on L3-L7 using an identity based security model that is decoupled from network addressing. Cilium can act as a highly scalable replacement for kube-proxy and offers deep observability through its optional Hubble component.
```

Comment:
Simplifying & adding hyphenation.
```suggestion
* [Cilium](https://github.com/cilium/cilium) is a networking, observability, and security solution with an eBPF-based dataplane. It provides a simple flat Layer 3 network that spans multiple clusters in either a native routing or overlay/encapsulation mode, and can enforce network policies on L3-L7 using an identity-based security model that is decoupled from network addressing. Cilium can act as a highly scalable replacement for kube-proxy and offers deep observability through its optional Hubble component.
```

Reply:
I'm happy to add the - but for the spanning multiple clusters, I would rather leave it as is because it is only an option while the suggestion implies that it is required

Reply:
Ah yes, that's fine!